### PR TITLE
Admin Page, Appointment and Patient Page Styling

### PR DESCRIPTION
**Pull request by Maxwell2Gyamfi**
In this PR I have managed to style the admin page and added functionalities such as add a new user, delete and updating an existing user. I have modified the databases script such that integrity is maintained when a user is deleted. The use of on delete cascade query will delete each row of a table where foreign key of delete user is present.

I have style the appointment and prescription page such that each action can be performed with just a click.
I have created the styling for reordering a prescription but it is not implement yet as assigned to Bredan.

I have managed to make appointment servlet thinner by simplifying the jsp file.

### CODE REVIEW

File: src/main/webapp/styles/appointment.css
Comment (CharlievWilliams) [inline src/main/webapp/styles/appointment.css:41]: My bad here mate. Could we please remove `blueAlternateButton` and `pinkAlternateButton` because I don't think they're in use anymore

Comment (CharlievWilliams) [reply src/main/webapp/styles/appointment.css:41]: I'm also seeing duplicate CSS in `Appointments.css` which is confusing me

File: src/main/webapp/appointments.jsp
Comment (CharlievWilliams) [inline src/main/webapp/appointments.jsp:16]: Ah is it because we have combined the two documents?

File: src/main/webapp/styles/main.css
Comment (CharlievWilliams) [inline src/main/webapp/styles/main.css:117]: Could we please rename the `blueButton` and `pinkButton` ID to something like `blueAppointmentButton` and `pinkAppointmentButton`? My IDE is giving me warnings because this ID has the same name as the original class.

File: src/main/webapp/styles/appointment.css
Comment (CharlievWilliams) [reply src/main/webapp/styles/appointment.css:41]: I've seen on slack we're removing this file! Could we delete it here then please and remove `blueAlternateButton` and `pinkAlternateButton` from `main.css`.

File: src/main/webapp/styles/appointment.css
Comment (CharlievWilliams) [inline src/main/webapp/styles/appointment.css:1]: I see this file is unused! Could we please delete this file and all commented out references to it.

File: src/main/webapp/styles/main.css
Comment (CharlievWilliams) [inline src/main/webapp/styles/main.css:225]: My bad. These classes are unused. Could we please delete blueAlternateButton and pinkAlternateButton.

File: src/main/webapp/styles/main.css
Comment (CharlievWilliams) [inline src/main/webapp/styles/main.css:256]: If we reference colours we should use the variables at the top of the file. Like:

`color: var(--secondaryDark);`

There's a few that need to be updated in this file.

File: src/main/java/Controllers/AppointmentServlet.java
Comment (CharlievWilliams) [inline src/main/java/Controllers/AppointmentServlet.java:10]: Correct me if I'm wrong.

Is the plan to revert changes to AppointmentServlet.java and Appointments.jsp?

File: src/main/java/Controllers/RegistrationServlet.java
Comment (CharlievWilliams) [inline src/main/java/Controllers/RegistrationServlet.java:64]: Is this supposed to still be commented out?

File: src/main/java/Models/User/UserAccountDAO.java
Comment (CharlievWilliams) [inline src/main/java/Models/User/UserAccountDAO.java:178]: Does `updateUserAccountDetails` have different functionality to `updateUserAccount` on line 138?

File: src/main/webapp/styles/main.css
Comment (CharlievWilliams) [reply src/main/webapp/styles/main.css:225]: Hmm, I see you have made some use of them actually.

File: src/main/webapp/dashboards/patient.jsp
Comment (CharlievWilliams) [inline src/main/webapp/dashboards/patient.jsp:24]: How come this bit is commented out?

Comment (CharlievWilliams) [reply src/main/webapp/dashboards/patient.jsp:24]: Same thing on admin.jsp.

File: src/main/java/Controllers/AppointmentServlet.java
Comment (Maxwell2Gyamfi) [reply src/main/java/Controllers/AppointmentServlet.java:10]: We won't be needing them as they will be deleted in the fat controllers branch

File: src/main/java/Controllers/RegistrationServlet.java
Comment (Maxwell2Gyamfi) [reply src/main/java/Controllers/RegistrationServlet.java:64]: Yes, it is, a button calls it

File: src/main/java/Models/User/UserAccountDAO.java
Comment (Maxwell2Gyamfi) [reply src/main/java/Models/User/UserAccountDAO.java:178]: updateUserAccountDetails is used to update user details in admin page excluding password
Update user removes user such as doctor, admin and nurse from pending list

File: src/main/webapp/dashboards/patient.jsp
Comment (Maxwell2Gyamfi) [reply src/main/webapp/dashboards/patient.jsp:24]: It wasn't showing the name from the servlet somehow, could be commented back once it works

**Review by CharlievWilliams — APPROVED** [review]
Happy to approve, pending Tom's approval.

File: src/main/java/Filters/DashboardFilter.java
Comment (Tom-Goring) [inline src/main/java/Filters/DashboardFilter.java:37]: For future reference, this kind of thing should be removed from commits before making PRs

**Review by Tom-Goring — APPROVED** [review]
LGTM!

**Pull request merged**
